src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -430,7 +430,7 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     const istioStatusMsgs = this.getStatusMessages();
     const objectReferences = this.objectReferences();
     const refPresent = objectReferences.length > 0;
-    const showCards = refPresent || this.hasOverview() || istioStatusMsgs.length > 0;
+    const showCards = refPresent || this.hasOverview() || (!!istioStatusMsgs && istioStatusMsgs.length > 0);
```

Comment:
Not sure if I understand this validation, `istioStatusMsgs` will be always an array, or perhaps I'm missing some case.

Reply:
It is pretty unexpected. Testing around, I came across a nasty bug. It was a length access to a undefined variable.